mautic/mautic
# Make the option “Block search engines from indexing this file” enabled by default for assets (Creating an asset)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

I believe that when you make a file available for download on Mautic you want the person to only have access to that file if they have filled out a form or performed the action you want, rather than letting search engines potentially have a direct link to download the material.
This change is about reducing the number of clicks needed to reach a common configuration.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to the create asset page. 
3. See the field "Block search engined from indexing this file is checked by default. 
![image](https://github.com/mautic/mautic/assets/1926222/64f18da1-e91b-44f6-8d7d-8f71c0220344)
